x-pack/test/functional_cors/plugins/kibana_cors_test/server/plugin.ts

```diff
@@ -78,8 +76,9 @@ export class CorsTestPlugin implements Plugin {
         return h.response(renderBody(kibanaUrl));
       },
     });
-    await server.start();
+    server.start();
```

Comment:
We're no longer waiting for the underlying server to start, but from some local testing, it doesn't seem to introduce flakiness to the associated test suite.